### PR TITLE
Favicon and meta properties

### PR DESCRIPTION
**Pull request by danielduhh**
### Description:

Updated the index.html meta properties to match those of https://www.capitolhillblockparty.com.
This includes the official favicon 👌 .

Screen Shot 2019-07-15 at 7 51 17 PM

### UI images:

No ui changes

### Unit Test Approach:

No tests

### Test Results:
- [ x ] Did you run `npm test` and did the results pass?
- [ ] Did you add unit tests that cover your changes? No tests needed

Describe other (non-unit) test results here.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -3,11 +3,27 @@
 <head>
   <meta charset="utf-8" />
   <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
-  <meta
-          name="viewport"
-          content="width=device-width, initial-scale=1, shrink-to-fit=no"
-  />
-  <meta name="theme-color" content="#000000" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"/>
+  <meta name="theme-color" content="#dd4e36" />
```
Comment (stephencerqueira) [inline public/index.html:7]: This doesn't have a pre-defined color?

**Review by cbessee — APPROVED** [review]
So official now. Looks great!

File: public/manifest.json [hunk 2]
```diff
@@ -10,6 +10,6 @@
   ],
   "start_url": "./index.html",
   "display": "standalone",
-  "theme_color": "#1d96dc",
-  "background_color": "#3d90b6"
+  "theme_color": "#dd4e36",
```
Comment (stephencerqueira) [inline public/manifest.json:13]: No pre-defined colors?

**Review by stephencerqueira — COMMENTED** [review]
See comments, otherwise LGTM

**Pull request merged**